doc/whatsnew/1.8.rst

```diff
@@ -276,5 +276,8 @@ Other Changes
     if 'bar' in (foo or {}):
       pass
 
+* Redefinition of dummy function is now possible. No ``function-redefined`` message will be emitted anymore if
```

Comment:
This one is very nitpicky (and I'm not a native speaker so I may be wrong), but:

_No message will be emitted_ seems slightly off comparing to _Message won't be emitted_.

Same goes to _if dummy functions are redefined_ - _**when** dummy functions are redefined_ seems slightly better.